blue-note/theme.json

```diff
@@ -267,13 +276,56 @@
 		},
 		"elements": {
 			"button": {
+				"spacing": {
+					"padding": {
+						"top": "calc(0.667em + 1px)",
+						"right": "calc(1.333em + 1px)",
+						"bottom": "calc(0.667em + 1px)",
+						"left": "calc(1.333em + 1px)"
+					}
```

Comment:
The button block has spacing that adjusts for the `outline` variation:
Because the border was changed from 2px to 1px, the padding also needed to be changed from +2px to +1px.